generated/nitclk/session.py

```diff
@@ -0,0 +1,1156 @@
+        triggers, configure_for_homogeneous_triggers does not affect script
+        triggers. Script triggers are considered to be configured for all
+        sessions if either one or the other of the following conditions are
+        true: - No session has a script trigger that is None - One session has a
+        script trigger that is None and all other sessions have script triggers
+        other than None. The one session with the None trigger must have
+        script_trigger_master_session set to itself, indicating
+        that the session itself is the script trigger master Pause Triggers
+        configure_for_homogeneous_triggers configures generation sessions
+        that support pause triggers to share them, if the pause triggers are
+        None (no trigger configured) for all except one session. The pause
+        triggers are shared by: - Implicitly exporting the pause trigger from
+        the session whose script trigger is not None - Configuring the other
+        sessions that support the pause trigger for digital-edge pause triggers
+        with sources corresponding to the exported pause trigger - Setting
+        pause_trigger_master_session to the session that is
+        exporting the trigger for all sessions that support script triggers If
+        the pause triggers are configured for all generation sessions that
+        support pause triggers, configure_for_homogeneous_triggers does not
+        affect pause triggers. Pause triggers are considered to be configured
+        for all sessions if either one or the other of the following conditions
+        is true: - No session has a pause trigger that is None - One session has
+        a pause trigger that is None and all other sessions have pause triggers
+        other than None. The one session with the None trigger must have
+        pause_trigger_master_session set to itself, indicating
+        that the session itself is the pause trigger master Note: TClk
+        synchronization is not supported for pause triggers on acquisition
+        sessions.
+
+        Args:
+            sessions (list of int): sessions is an array of sessions that are being synchronized.
+
+        '''
+        session_count_ctype = _visatype.ViUInt32(0 if sessions is None else len(sessions))  # case S160
+        sessions_ctype = get_ctypes_pointer_for_buffer(value=_converters.convert_to_nitclk_session_num_list(sessions), library_type=_visatype.ViSession)  # case B520
+        error_code = self._library.niTClk_ConfigureForHomogeneousTriggers(session_count_ctype, sessions_ctype)
+        errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
+        return
+
+    def finish_sync_pulse_sender_synchronize(self, sessions, min_time):
+        r'''finish_sync_pulse_sender_synchronize
+
+        TBD
+
+        Args:
+            sessions (list of int): sessions is an array of sessions that are being synchronized.
```

Comment:
needs converted (can be addressed in separate PR)

Reply:
What do you mean by "needs converted"?

Documentation will be cleaned up in a later PR, but only what is generated, not in the content itself. So "list of int" will change but "sessions is an array of sessions that are being synchronized." will not

Reply:
I meant "needs a converter".